### PR TITLE
Update virtual-fields.rst

### PR DESCRIPTION
**Pull request by EOM**
Add example (Paginator + JOINS + virtualField) on fly in Controller

### CODE REVIEW

File: en/models/virtual-fields.rst [hunk 1]
```diff
@@ -120,6 +120,76 @@ best to define the virtualFields in your model's constructor::
 This will allow your virtualFields to work for any alias you give a
 model.
 
+Pagination and Virtual fields set in controller with JOINS
+=============================================================
+The following example allows to have a counter of a hasMany association and allows to use without problem
```
Comment (markstory) [inline en/models/virtual-fields.rst]: I think this sentence is missing a you.

Comment (EOM) [inline en/models/virtual-fields.rst]: No problem, my English is very bad.
I helped 100% with Google Translate.
With that you understand my text and your correction correct that concept to a bleated English, it is perfect for me. 👍 
Thank you.

Comment (markstory) [inline en/models/virtual-fields.rst]: Your english is much better than my Spanish :smile:

Comment (EOM) [inline en/models/virtual-fields.rst]: Hi @markstory
What happens I do not see the change in the Doc ..?
I have understood that my example was accepted for the CakePHP Doc.

Comment (markstory) [inline en/models/virtual-fields.rst]: I'll fix up the missing 'you' post merge.

**Pull request merged**